applications/nrf5340_audio/src/bluetooth/bt_management/scanning/bt_mgmt_scan.c

```diff
@@ -16,6 +16,20 @@ LOG_MODULE_REGISTER(bt_mgmt_scan, CONFIG_BT_MGMT_SCAN_LOG_LEVEL);
 
 static char srch_name[BLE_SEARCH_NAME_MAX_LEN];
 
+static void addr_print(void)
+{
+	int ret;
+	char dev[BT_ADDR_LE_STR_LEN];
+	static struct bt_le_oob _oob = {.addr = 0};
+
+	/* This call will create a new RPA */
+	ret = bt_le_oob_get_local(BT_ID_DEFAULT, &_oob);
+	__ASSERT(ret == 0, "bt_le_oob_get_local failed");
+
+	(void)bt_addr_le_to_str(&_oob.addr, dev, BT_ADDR_LE_STR_LEN);
+	LOG_INF("Local RPA: %s. May time out. Updates not printed", dev);
```

Comment:
Since this might time out and updates are not printed, maybe it is not necessary to print the RPA on the scanning side?

Reply:
You need to, or else it will not be possible to check that one has connected to the correct kit. But I do see your point.